circle.yml

```
@@ -249,6 +247,24 @@ jobs:
           paths:
             - ~/build/corpus
 
+  spectest:
+    executor: linux-gcc-9
+    steps:
+      - run:
+          name: "Download spectest files"
+          command: |
+            git clone https://github.com/wasmx/wasm-spec --branch vanilla-json --single-branch
+      - attach_workspace:
+          at: "~"
+      - run:
+          name: "Run spectest"
+          command: |
+            set +e
+            expected="  PASSED 2857, FAILED 142, SKIPPED 8814."
+            result=`~/bin/fizzy-spectests --skip-validation wasm-spec/test/core/json | tail -1`
```

More portable (and more readable IMHO).
```suggestion
            result=$(~/bin/fizzy-spectests --skip-validation wasm-spec/test/core/json | tail -1)
```

---

Fixed.